aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
i have created a new branch with a corresponding PR in which we can discuss further actions.